### PR TITLE
Add image credits input

### PR DESCRIPTION
**Pull request by mikachan**
This PR adds a textarea input for adding image credits to a theme's readme.txt file.

Textarea:
image

Readme.txt output:
image

### Testing Instructions
1. Go to the Create Block Theme admin page
2. Create a child theme, clone a theme, or create a blank theme
3. There should be a new textarea near the bottom of the theme metadata form for image credits
4. Try adding some image credits to this textarea
5. When you generate the theme, the image credits content should be added to the end of the readme.txt file

### CODE REVIEW

File: admin/create-theme/theme-form.php [hunk 1]
```diff
@@ -163,6 +163,29 @@ public static function create_admin_form_page() {
...
 								<br /><br />
+								<label>
+									<?php _e( 'Image Credits:', 'create-block-theme' ); ?><br />
+									<small><?php _e( 'List the credits for each image you have included in the theme. Include the image name, license type, and source URL.', 'create-block-theme' ); ?></small><br />
+									<small>
+										<?php
+										printf(
+											/* Translators: Bundled resources licenses link. */
+											esc_html__( 'All bundled resources must have GPL-compatible licenses (%s).', 'create-block-theme' ),
+											'<a href="' . esc_url( __( 'https://make.wordpress.org/themes/handbook/review/resources/#licenses-bundled-resources', 'create-block-theme' ) ) . '">read more</a>'
```
Comment (madhusudhand) [inline admin/create-theme/theme-form.php]: add attribute `target="_blank"`

**Review by madhusudhand — COMMENTED** [review]
It is working as advertised.

Created [this issue](https://github.com/WordPress/create-block-theme/issues/381) for the issue where the credits are removed from theme clone.

File: admin/create-theme/theme-form.php (hunk 1 above)
Comment (mikachan) [reply admin/create-theme/theme-form.php]: Good idea! I've added this in https://github.com/WordPress/create-block-theme/pull/380/commits/34df4329e2bf719faf2185d7122271dd82b90298, I also added it to the 'read more' link in the theme tags section.

**Comment by mikachan** [comment]
Thanks @madhusudhand!

> Created https://github.com/WordPress/create-block-theme/issues/381 for the issue where the credits are removed from theme clone.

I noticed this too - I've been working on a fix, hopefully I'll have a PR ready soon.

**Review by madhusudhand — APPROVED** [review]
LGTM!

**Pull request merged**